cmd/clusterctl/clusterdeployer/clusterdeployer.go

```diff
@@ -159,6 +160,10 @@ func (d *ClusterDeployer) Delete(targetClient clusterclient.Client) error {
 		return errors.Wrap(err, "unable to pivot Cluster API stack to bootstrap cluster")
 	}
 
+	// TODO: Find a better way to validate that the pivoted objects have been reconciled before deleting
+	klog.Info("Sleeping for 5 minutes to wait for pivoted objects to be reconciled")
+	time.Sleep(5 * time.Minute)
```

Comment:
Should we make this a variable? 

Seems a long sleep timeout, could we maybe track the clusters' modified timestamp instead?

Reply:
Instead of sleep, I'm trying to poll the resources until they all have a status.